### ISSUE TITLE
is there a way to use nginx in front of a cf node server that uses nodejs-buildpack

### ISSUE THREAD

**Issue by dagda1** [cloudfoundry/nodejs-buildpack#155]
I would like to put nginx in front of the cf node server to allow me to configure a web server without having to write code.

At the moment, if I want to add gzip for example, I have to add the dependency and write some code.  If I had nginx in front of the node server then I can just configure nginx.

Is there a way of doing this?

**Comment by sclevine** [comment]
To clarify, I think you want to use NGINX as a reverse-proxy in front of a Node.js HTTP server. Is this correct?

**Comment by dagda1** [comment]
@sclevine Yes that is absolutely correct.  In the past I've used docker-compose to do this and my ngninx.conf looks like this:

```
        location  / {
            proxy_set_header Host $http_host;
            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
            proxy_set_header X-Forwarded-Proto https;
            proxy_set_header X-Real-IP $remote_addr;
            proxy_set_header X-Scheme $scheme;
            proxy_redirect off;
            proxy_pass   http://server:3000;
        }
```

`http://server:3000` would be my node process but I have no idea how to do this with the node buildpack.

**Comment by dagda1** [comment]
@sclevine Thank you, do you have any links or anything for container to container networking?  I have not found anything in the docs about this.

**Comment by sclevine** [comment]
https://docs.cloudfoundry.org/concepts/understand-cf-networking.html

**Comment by RubensTen** [comment]
Hi @tylerphelan, I have the similar issue, what was your best implemetation/solution ?
do you have any example ?
